tests/unit/BitOasis/Coin/Address/LitecoinAddressTest.php

```diff
@@ -59,4 +59,14 @@ protected static function getCurrency() {
 		return UnitTestUtils::getCryptocurrency(Cryptocurrency::LTC);
 	}
 
+	/**
+	 * @param string $legacyFormat
+	 * @param string $newFormat
+	 * @dataProvider providerToNewAddressFormat
+	 */
+	public function testLegacyAddress($legacyFormat, $newFormat) {
+		$litecoinAddress = new LitecoinAddress($newFormat, self::getCurrency());
+		$this->assertEquals($litecoinAddress->getLegacyAddress(), $legacyFormat);
```

Comment:
`Lciocvp1PvQyMH9Srmxn7dSC94Gk2YtCLm` is not legacy format of `Lciocvp1PvQyMH9Srmxn7dSC94Gk2YtCLm` address. `Lciocvp1PvQyMH9Srmxn7dSC94Gk2YtCLm` has no legacy format.

Reply:
In that case it should be returning null.